app/expenses/[expenseId]/decide-modal.tsx

```diff
@@ -0,0 +1,47 @@
+import { useLocalSearchParams } from 'expo-router';
+import React, { useContext, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { BackHandler } from 'react-native';
+
+import ActivityDecisionPopover from '@/components/ui/popover/ActivityDecisionPopover';
+import { GlobalContext } from '@/context/GlobalContext';
+import useAddExpenseDecision from '@/hooks/expense/UseAddExpenseDecision';
+
+export default function ExpenseDecideModal() {
+  const { t } = useTranslation();
+
+  const params = useLocalSearchParams<{ expenseId: string }>();
+  const { userData } = useContext(GlobalContext);
+  const [message, setMessage] = useState<string | undefined>(undefined);
```

Comment:
dont sure if we want to use undefined and if not better approach will be using empty string sth
```tsx
    const [message, setMessage] = useState<string>(''); 
    const [decision, setDecision] = useState<'ACCEPT' | 'REJECT' | ''>('');

    const { mutate: addExpenseDecision, isPending: iseDecisionPending } = useAddExpenseDecision({
      expenseId: params.expenseId,
      groupId: userData.currentGroupId || '', 
      decision: decision || '', 
      message: message,
    });

...

onAcceptPress={() => setDecision(decision === 'ACCEPT' ? '' : 'ACCEPT')}
onRejectPress={() => setDecision(decision === 'REJECT' ? '' : 'REJECT')}

```

or like as it is now

Reply:
or with one more type as 'NONE'